blog/2024-09-18-case-study-falai/index.md

```diff
@@ -0,0 +1,121 @@
+them met their reliability and performance needs. Low throughput, sluggish
+downloads, and intermittent 500 errors made it impossible to guarantee their
+diffusion endpoints could process requests in real time. After partnering with
+Tigris, fal.ai didn’t need to choose between performance, reliability, and cost.
+
+> have been using @TigrisData at @fal for the last 2 months. ingested 10s of TBs
+> of data in mere hours while storing 100 TB+ without any hassle. much much more
+> reliable than anything else we have used. and also MUCH FASTER. i am impressed
+> honestly. thanks @martin_casado for rec
+> —[Batuhan Taskaya](https://x.com/isidentical/status/1792631256586338349?s=46&t=uiM9vzvUExQcdhsrC5jgfg),
+> Head of Engineering at fal.ai
+
+## A hub for developers
+
+Image generation, video generation, upscaling images, speech-to-text,
+text-to-speech, all sorts of media related functionality— keeping up with the
+latest and greatest models and tools is a lot for any developer. And it’s often
+unclear which models can keep up with a production deployment. fal.ai is a
+unified hub for integrating these models as reliable utilities. They’ve already
+selected the best model, and delivered it in the fastest way possible.
+
+> You wanna go to production? Let's go to production. --Burkay Gur, Co-founder
+> fal.ai
+
+And when we say “latest and greatest,” we mean it. [fal.ai](http://fal.ai)
+offers the largest SOTA open source text-to-image model to date,
+[Flux](https://blog.fal.ai/flux-the-largest-open-sourced-text2img-model-now-available-on-fal/).
+Built by the original Stable Diffusion team, fal.ai was their first choice for
+digging into the model and optimizing it to run on a real-world production grade
+endpoint.
+
+Dev friendly features like one-click fine tuning make it unbelievably easy to
+customize models to your users without sacrificing on inference speeds.
+[fal.ai](http://Fal.ai) also exposes raw WebSockets to make it easy to reduce
+overall latency and simplify the developer experience. Getting whichever GPU is
+closest and cheapest has never been easier.
+
+## Unlimited horizontal scaling at an 85% discount with Tigris
+
+Sending data around so many GPUs across so many clouds could lead to a
+dizzyingly high egress bill. Other zero egress cost storage providers struggle
+to meet the stringent performance requirements of modern platforms. Finding a
+storage solution built for an extremely performant global deployment was
+essential to keeping fal.ai accessible to a broad range of developers. Not only
+was Tigris
+[“just crazy fast,”](https://x.com/isidentical/status/1817637355366613374?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1817637355366613374%7Ctwgr%5E8024e11d092aa48518bdab2d4fb51742457694f6%7Ctwcon%5Es1_c10&ref_url=https%3A%2F%2Fpublish.twitter.com%2F%3Furl%3Dhttps%3A%2F%2Ftwitter.com%2Fisidentical%2Fstatus%2F1817637355366613374)
+it saved fal.ai 85% on their object storage costs as compared to other clouds
```

Comment:
@ovaistariq is there a way to make a bigger callout for 85% cost savings in docusaurus?

Reply:
I also think this is fine to merge and something we can style when we create a custom page.

Reply:
Sounds good.

Reply:
I have just made it bold, slightly better.